### ISSUE TITLE
Mac OS segfaulting

### ISSUE THREAD

**Issue by acbart** [pvcraven/arcade#513]
## Bug Report
Arcade cannot be imported (and programs using Arcade do not work, by extension) without segfaulting. This seems to be on newer macs, and I suspect it's related to the new update.

### Actual behavior:

When my students `import arcade` in the python console, it prints out something like:

```
Start
Start
Segfault
```

I can't recall the exact text, but will try to update with more information when I get a test machine.

### Expected behavior:

Not segfaulting.

### Steps to reproduce/example code:

We pip installed the latest version of arcade

```
pip install arcade
```

```
import arcade
```

Let me know if there's more information you'd like, I can start collecting from students in the wild.

**Comment by pvcraven** [comment]
Does it segfault with a simple open window, but not draw anything?

http://arcade.academy/examples/starting_template.html

**Comment by pvcraven** [comment]
If you are familiar with using the debugger (pycharm makes it pretty easy) you might be able to narrow down where it crashes.

**Comment by acbart** [comment]
It segfaulted for them when they did `import arcade`, even before they tried to open a window or do anything else. I've reached out to see if I can get them to run some code to narrow down when the segfault occurs - haven't head back from them yet.

**Comment by acbart** [comment]
They are able to get Pyglet code running to create a window, so it's probably something more mundane in the Arcade side. I have general installation day this Wednesday, so I'll have a lot more bug reports then, most likely. Or maybe it'll go smoothly and my other students will have been an outlier.

**Comment by acbart** [comment]
Had a student today that I could actually grab some debug info for.

System info:
ProductName: Mac OS X
ProductVersion: 10.15.1
BuildVersion: 19B88

This is Python 3.7

And here's a stack trace:

https://pastebin.com/raw/6NyDtJmC

Any insight would be greatly appreciated!

**Comment by acbart** [comment]
I have some students with Catalina that are able to run things, at least in Thonny. They sometimes get weird errors when they close the application, but that doesn't prevent them from importing and running arcade.

For the student that let me generate that stack trace above, I tried a few different things (installing Python 3.8, reinstalling Thonny, restarting). I'm not sure if one of them made a difference, but suddenly I've been able to get Pyglet windows to appear when I run their examples directly. I still can't import arcade, but some trial and error revealed that the error seems to be when I do:

```
import pyglet.input
```

Could this somehow be related to joystick type of things? I have another section about to come in, so I'll see if I have more test cases to play with.
